docs/explanations/how_to_interpret_output_of_pro_vulnerability_show.rst

```diff
@@ -0,0 +1,152 @@
+How to interpret the output of pro vulnerability show
+*****************************************************
+
+If you suspect that a CVE affects your system and wants to verify
+if that is true, you can use the ``pro vulnerability show`` command for this.
+This command will show you all the related information to a CVE **if** it affects
+your system.
+
+For example, let's assume that ``CVE-2022-2286`` affects your system and you run:
+
+.. code-block:: bash
+
+    pro vulnerability show CVE-2022-2286
+
+You will see an output similar to this one:
+
+.. code-block:: text
+
+   ● CVE-2022-2286
+   Public URL: https://ubuntu.com/security/CVE-2022-2286          
+   Published at: 2022-07-02 19:15:00
+   Ubuntu vulnerability data published at: 2024-10-22 23:37:41
+   APT package information updated at: 2024-10-23 19:37:11
+   Ubuntu priority: low
+   CVSS score: 7.8
+   CVSS severity: high
+   
+   [DESCRIPTION]
+   Out-of-bounds Read in GitHub repository vim/vim prior to 9.0.
+   
+   [AFFECTED INSTALLED PACKAGES]
+   NAME         STATUS  CURRENT VERSION        FIXED BY                     FIX AVAILABLE FROM
+   vim          fixed   2:7.4.1689-3ubuntu1.5  2:7.4.1689-3ubuntu1.5+esm19  esm-infra
+   vim-common   fixed   2:7.4.1689-3ubuntu1.5  2:7.4.1689-3ubuntu1.5+esm19  esm-infra
+   vim-runtime  fixed   2:7.4.1689-3ubuntu1.5  2:7.4.1689-3ubuntu1.5+esm19  esm-infra
+   vim-tiny     fixed   2:7.4.1689-3ubuntu1.5  2:7.4.1689-3ubuntu1.5+esm19  esm-infra
+   
+   [RELATED USNs]
+   VULNERABILITY  TITLE                AFFECTED INSTALLED PACKAGES
+   USN-6270-1     Vim vulnerabilities  vim
+
+Let's break it down the output of this command. We start by telling you some basic information for
+the CVE:
+
+* **Public URL**: The ubuntu security page for this CVE
+* **Published at**: The published date for the CVE
+* **Ubunut vulnerability data published at**: The published for the vulnerability data we consume to
+  generate this output
+* **APT package information updated at**: The last time the APT state was updated in the system
+  (i.e. running an apt install operation)
+* **Ubuntu priority**: The ubuntu priority for this CVE
+* **CVSS score**: The CVSS score of the CVE
+* **CVESS severity**: The CVSS severity of the CVE
+
+After that, we display the CVE description and related notes (if the CVE has notes).
+
+The next block is now displaying which installed packages in the machine are affected by the CVE.
+Whe are using table to display those affected packages where:
+
+* **NAME**: The package name
+* **STATUS**: The CVE status for this package
+* **CURRENT VERSION**: The current installed version of the package
+* **FIXED BY**: The version of that package that fix the CVE issue
+* **FIX AVAILABLE FROM**: The pocket where the fix version can be found on
+
+Finally, we also display the related USNs to the CVE, in case you want to take a look on them as
+well.
```

Comment:
```suggestion
Finally, we also display any USNs related to the CVE.
```

Reply:
Should we link here to the page about related USNs?

Reply:
No, this is a different concept. Our related USNs documentation tell about a relation between USNs and other USNs, no CVEs to USNs

Reply:
Yeah you're right, I was having a moment lol